Doc/library/dataclasses.rst

```diff
@@ -461,8 +461,8 @@ Module contents
 
 .. function:: is_dataclass(obj)
 
-   Return ``True`` if its parameter is a dataclass or an instance of one,
-   otherwise return ``False``.
+   Return ``True`` if its parameter is a dataclass (including subclasses of a dataclass)
+   or an instance of one, otherwise return ``False``.
```

Comment:
```suggestion
   Return ``True`` if its parameter is a dataclass (including subclasses of a
   dataclass) or an instance of one, otherwise return ``False``.
```